src/components/custom/_signup-new.tsx

```diff
@@ -276,8 +246,7 @@ const SignupNew = ({
                     type="button"
                     social
                 >
-                    <img src={Google} alt="google" width="24" height="24" />
-                    <SocialText>Google</SocialText>
+                    <img src={Google} alt="google" width="37" height="37" />
```

Comment:
Why do we have `width` and `height` 37 on this and 36 on other icons? Can we have `width` and `height` 36 on this as well?

Reply:
Because the google svg icon is smaller than the others, that's why i make the size different 😶

Reply:
you need to request a new image from PD team. We should standardize the size of icons.